nam/train/gui.py

```diff
@@ -191,6 +198,30 @@ def __init__(self):
 
         self._check_button_states()
 
+    def getAdditionalOptionsFrames(self):
```

Comment:
Nit: can we make this function name PEPpy & private in scope? (`_get_additional_options_frames` plz)